components/src/util/util.js

```diff
@@ -0,0 +1,23 @@
+
+export function toPromise (fn) {
+  return new Promise((resolve, reject) => {
+    fn((err, res) => {
+      if (err) {
+        reject(err);
+      } else {
+        resolve(res);
+      }
+    });
+  });
+}
+
+export function stringifyIfObject (any) {
```

Comment:
yes it was migrated
removed it now anyway

will remove it form status page as well, it's not really needed